InsertionSort.java

```diff
@@ -0,0 +1,27 @@
+public class MyInsertionSort {
```

Comment:
This class must implement the ISortingAlgorithm interface.
Like it was done with default algorithm.